Makefile

```diff
@@ -88,7 +91,13 @@ sdist:
 	$(SA) $(ENV) && python setup.py sdist $(POST_SDIST) \
 		&& rm -rf *.egg-info
 
-dist: lint bdist sdist kernelspecs ## Make source, binary and kernelspecs distribution to dist folder
+helm-chart:
```

Comment:
Isn't this command available to run alone? If it is, adding description will be helpful to print it out as a result of `make help`